project/frontend/src/containers/CreateCompetition/CreateCompetition.js

```diff
@@ -26,18 +39,38 @@ const CreateCompetition = (props) => {
           <div className={classes.AddParticipants}>
             <p>
               {" "}
```

Comment:
What's this?

Reply:
Idk either lol VS code added it for me when I saved the file but I removed all of it now. :)